src/js/angular/rest/export.rest.service.js

```diff
@@ -0,0 +1,50 @@
+angular
+    .module('graphdb.framework.rest.export.service', [])
+    .factory('ExportRestService', ExportRestService);
+
+ExportRestService.$inject = ['$http', '$repositories', '$translate'];
+
+const REPOSITORIES_ENDPOINT = 'repositories';
+function ExportRestService($http, $repositories, $translate) {
+    return {
+        getExportedStatementsAsJSONLD
+    };
+
+    function getExportedStatementsAsJSONLD(context, repo, graphsByValue, acceptHeader, linkHeader, auth, forSelectedGraphs) {
```

Comment:
Document and reorder these parameters a bit. It'd have been better if they were fewer but still.